src/main/resources/messages_en.properties

```diff
@@ -122,6 +122,7 @@ invalid.observation.id=Invalid observation id.
 invalid.observation.value=Invalid value.
 dataset.name.empty.name=Dataset name should not be empty.
 instance.does.not.exist=Instance does not exist.
+instance.already.exists=Instance number {0} already exists.
```

Comment:
We won't need this validation when we don't expose the instance number as parameter

Reply:
Done